front/app/components/ConsentManager/Container.tsx

```diff
@@ -65,18 +58,19 @@ const Container = ({
     if (isConsentRequired && !isEmpty) {
       setIsCancelling(true);
     } else {
-      setIsDialogOpen(false);
+      setConsentModalView('main');
```

Comment:
The cancel button in the preferences view should just take us back to the main cookie modal view (not close the modal entirely).